packages/zosfiles/src/api/utils/ZosFilesUtils.ts

```diff
@@ -197,7 +196,7 @@ export class ZosFilesUtils {
         dataSetName: string,
         returnMessage: string,
         hsmCommand: any,
```

Comment:
I believe `hsmCommand` will also be of type IZosFilesUtils?
Maybe I'm confused.

Reply:
@zFernand0 `hsmCommand` here refers to the request type, so for example the hDelete API would have the request type `{ request: "hdelete" }`  https://github.com/zowe/zowe-cli/blob/84280b7837ea76af65cb02f24cdef0c5728f428a/packages/zosfiles/src/api/methods/hDelete/HDelete.ts#L45

Whereas the hRecall API would send the request `{ request: "hrecall" }` to the base handler `dfsmsHsmCommand` https://github.com/zowe/zowe-cli/blob/84280b7837ea76af65cb02f24cdef0c5728f428a/packages/zosfiles/src/api/methods/hRecall/HRecall.ts#L45